proj3_chess/player.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+
+class Player:
+    def __init__(self, direction: int, symbol: Callable[[str], str]) -> None:
+        """This class represents a player in a chess game. \n
```

Comment:
Documentation for what the class represents should be under the class name (like in `ChessPiece`). In this case, because it is a constructor, method documentation is ok but do it in `reST` format (read about it)